api/integrations/git/post-receive

```diff
@@ -51,9 +52,11 @@ client = zulip.Client(
 
 # check_output is backported from subprocess.py in Python 2.7
 def check_output(*popenargs, **kwargs):
+    # type: (*Any, **Any) -> Any
     if 'stdout' in kwargs:
         raise ValueError('stdout argument not allowed, it will be overridden.')
-    process = subprocess.Popen(stdout=subprocess.PIPE, *popenargs, **kwargs)
+    process = subprocess.Popen(stdout=subprocess.PIPE, *popenargs, **kwargs) # type: ignore
+    # ^ Maybe too many arguments (3 instead of 2)? https://chat.zulip.org/#narrow/near/72687/stream/GCI.20help/topic/mypy
```

Comment:
The thing that confuses me is that I think the code was copied directly from Python core, so it may be that Python actually lets you do this?  This code definitely works...